docs/alerts/monitors/create-monitor.md

```diff
@@ -250,6 +233,23 @@ Log monitor triggers are evaluated by balancing the requirement of timely alert
 
 <img src={useBaseUrl('img/monitors/metrics-query.png')} alt="metrics query.png" style={{border: '1px solid gray'}} width="600" />
 
+#### Prerequisites
+
+To fully leverage AI-driven alerts for metrics monitors, you'll need:
+
+* **Automation Service**. Required for linking playbooks to metrics-based monitors.
+* **Metrics data**. Metrics data must be sent to Sumo Logic for anomaly detection.
```

Comment:
It might be helpful to suggest the amount of data required - I think consistent 30d data would be optimal, but I'm unsure what the minimum is. I can check with the team if we think it would be good to add in.

Reply:
@banant , can you speak to Andrew's feedback?